### PR TITLE
feat: added text-toast component

### PR DESCRIPTION
**Pull request by cscheffauer**
# Description

- Added Text Toast component as defined in the Figma link below. This is a precondition for the TemporaryUnmute toast.
- Updated tokens dependency to get latest tokens

# Links

Figma: https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=44750%3A103463

### CODE REVIEW

File: src/components/TextToast/TextToast.types.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   * Default: `center`
+   */
+  textAlignment?: TextToastTextAlign;
+
+  /**
+   * IconProps to be passed in - allows defining the icon which should
+   * be shown on the left-hand side of the text. At least the `name` property
+   * should be passed in through the object, otherwise the Icon will not be shown.
+   *
+   * Default: `undefined`
```
Comment (InteractiveTimmy) [inline src/components/TextToast/TextToast.types.ts:24]: same as above.

File: src/components/index.ts [hunk 2]
```diff
@@ -53,3 +53,4 @@ export { default as ListHeader } from './ListHeader';
 export { default as LoadingSpinner } from './LoadingSpinner';
 export { default as Tag } from './Tag';
 export { default as MeetingContainer } from './MeetingContainer';
+export { default as TextToast } from './TextToast';
```
Comment (InteractiveTimmy) [inline src/components/index.ts:56]: Are you missing the export on `src/index.js`?

**Review by InteractiveTimmy — CHANGES_REQUESTED** [review]
A few changes to improve refactor protection, some around JSDoc, and then some around style assignments.

Only real blocking change is that it's not exported at the root `src/index.js` file.

File: src/components/TextToast/TextToast.constants.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { TextToastTextAlign } from './TextToast.types';
+
+const CLASS_PREFIX = 'md-text-toast';
+
+const DEFAULTS = {
+  ALIGNMENT: 'center' as TextToastTextAlign,
```
[the remark this answers is not in the document]
Comment (InteractiveTimmy) [reply src/components/TextToast/TextToast.constants.ts:6]: This will prevent the need for type-casting.

**Review by InteractiveTimmy — CHANGES_REQUESTED** [review]

**Review by torgeadelin — APPROVED** [review]
Looking great!

**Pull request merged**